### DISCUSSION TITLE
Was that noticed? | Starship for Fedora served on Fedora Copr repository platform

### DISCUSSION THREAD

**Discussion by Ricky-Tigg** [starship/starship#5277] [Q&A]
Hello. This application for _Fedora Linux_ is served on _Fedora Copr_ repository platform, published by Atim. It is worth a mention at https://starship.rs/#quick-install. **Installation instructions:** `dnf -y copr enable atim/starship && dnf -y install starship`.

**Comment by davidkna** [comment]
We are already suggesting that COPR for CentOS, but Fedora should probably be listed alongside now that Fedora no longer packages starship.

**Comment by Ricky-Tigg** [comment]
_CentOS_? It has been discontinued in its traditional form; that's an IBM Red Hat's story.  If valid, a such suggestion would make sense for _Rocky Linux_.

**Reply by davidkna** [reply]
Given that it is hosted on COPR, I would prefer to solely mention the operating systems that are officially supported.
